### DISCUSSION TITLE
How to include and enable GLTF extensions with modelviewer?

### DISCUSSION THREAD

**Discussion by henrik-dons** [google/model-viewer#4903] [Q&A]
Hi all,

I'm reaching out to ask the possibilities of adding extensions from gltf-transform (open-source), I'll be adding a few that's would extend features and functionality here but you can check out others. 

https://gltf-transform.dev/extensions 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsDispersion 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsVolume 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsTransmission 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsIOR 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsClearcoat 
https://gltf-transform.dev/modules/extensions/classes/KHRMaterialsSheen 
https://gltf-transform.dev/modules/extensions/classes/KHRMeshQuantization 
https://gltf-transform.dev/modules/extensions/classes/KHRDracoMeshCompression

Let me know if this is possible already, or if we can see this added in the future!

**Comment by elalish** [comment]
We support rendering all of these and more already. Just drop your files into our editor to preview them and make sure all is as expected - it'll also let you know if there are glTF validation errors. If you scroll down through [this example](https://modelviewer.dev/examples/scenegraph/#pickMaterialExample), you'll even see how you can set textures and properties for these extension using JS in MV itself.
